app/models/concerns/oregon_digital/generic_metadata.rb

```diff
@@ -6,7 +6,8 @@ module GenericMetadata
+    PROPERTIES = %w[abstract accepted_name_usage access_restrictions accession_number acquisition_date alternative arranger art_series artist author award award_date barcode biographical_information box box_name box_number based_near canzoniere_poems cartographer citation collected_date collector common_name compass_direction composer contents contributor conversion copy_location copyright_claimant cover_description coverage creator creator_display cultural_context current_repository_id date date_digitized dedicatee description description_of_manifestation designer donor editor ethnographic_term event exhibit extent family file_size folder_name folder_number form_of_work format former_owner genus gps_latitude gps_longitude has_finding_aid has_part has_version higher_classification hydrologic_unit_code identification_verification_status identifier illustrator inscription institution interviewee interviewer isPartOf is_version_of issued item_locator keyword language layout legal_name license local_collection_id local_collection_name location location_copyshelf_location longitude_latitude_identification lyricist material measurements military_branch military_highest_rank military_occupation military_service_location mode_of_issuance mods_note object_orientation oembed_url order original_name_usage owner patron photographer phylum_or_division physical_extent place_of_production primary_set print_maker provenance publication_place publisher ranger_district recipient relation replaces_url repository resource_type rights_holder rights_statement scientific_name_authorship scribe series_name series_number set source source_condition species specimen_type sports_team state_or_edition street_address style_or_period subject taxon_class technique temporal tgn transcriber translator tribal_classes tribal_notes tribal_terms tribal_title use_restrictions view_date water_basin workType].freeze
+    CONTROLLED = %w[format_label]
```

Comment:
There's got to be a better way to specify this. `format_label` isn't a property like the others. The property itself has the `class_name` attribute, and there's also `self.controlled_properties` where format is included. Can one of those be used for the access later?

Reply:
This definitely wasn't the "Final solution" for this piece of code. This was to make it flow naturally through the rest of our code with minimal changes. But we should have a discussion about controlled vocabularies on the show page. Do we want them to be displayed as the search and external links? Do we want this to be different for each? We can add format_label to the properties which would be totally fine, but we need to decide how we would like to display these fields on the show page.